test/fuzz/mmap_alloc.cc

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2021-2022 Intel Corporation
+ */
+
+#include <cne_mmap.h>
+#include <string.h>
+/* Prototype required to fix "no previous prototype for function" error */
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
+
+/* When launching the mmap_alloc fuzz test, the -rss_limit_mb=N parameter
+ * needs to be passed. Otherwise, an out of memory error will happen.
+ */
+extern "C" int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    uint32_t bufcnt, bufsz;
+    mmap_type_t hugepage;
+    mmap_t *mm;
+
+    if (size > sizeof(bufsz))
+        size = sizeof(bufsz);
```

Comment:
The bufsz variable is only used to calculate the value of size when the argument passed in was > 4. Change the sizeof(bufsz) to sizeof(uint32_t)

If size > 4 bytes (sizeof(uint32_t)) then size becomes 4. Can the size value be zero, if so then we can just leave the routine.